packages/components/src/List/List.component.js

```diff
@@ -95,61 +107,66 @@ ListToolbar.propTypes = {
 }
  <List {...props}></List>
  */
-function List({
-	displayMode,
-	id,
-	list,
-	toolbar,
-	defaultHeight,
-	rowHeight,
-	getComponent,
-	components = {},
-	rowRenderers,
-}) {
-	const classnames = classNames('tc-list', theme.list);
-	const injected = Inject.all(getComponent, omit(components, ['toolbar', 'list']));
+class List extends React.Component {
```

Comment:
💬As we move forward to react 16.8, hooks & co, do we want to go more on Fonctional components ?
I think here you move the functional to class to use the static but do we want to still use classes ?

Reply:
Oups need to rollback on this, was something before hooks that i've done ;)